meilisearch-http/src/routes/indexes/documents.rs

```diff
@@ -151,10 +149,10 @@ pub async fn get_all_documents(
         .documents(path.into_inner(), offset, limit, attributes_to_retrieve)
         .await?;
 
-    debug!("returns: {:?}", documents);
-    Ok(HttpResponse::Ok().json(json!(
-        { "limit": limit, "offset": offset, "total": total, "results": documents }
-    )))
+    let ret = PaginationView::new(offset, limit, total as usize, documents);
+
+    debug!("returns: {:?}", ret);
```

Comment:
```suggestion
```

Reply:
Found the source;
https://github.com/meilisearch/transplant/issues/193
> warning In DEBUG level the Search endpoint should log the request body and response body.